python/perf-kernels/fused_moe/moe-gemm.py

```diff
@@ -482,12 +697,18 @@ def parse_args():
     parser.add_argument("-E", type=int, default=0, help="Number of experts")
     parser.add_argument("-top_k", type=int, default=0, help="top_k experts per token")
     parser.add_argument("-routed_weight", action='store_true', default=False)
+    parser.add_argument("-int8_w8a16", action='store_true', default=False)
+    parser.add_argument("-fp8_w8a8", action='store_true', default=False)
     parser.add_argument("-dtype", default='fp16')
+    parser.add_argument("-fp8_type", default='e5m2')
     args = parser.parse_args()
     return args
 
 
-arg_to_torch_dtype = {'fp16': torch.float16, 'bf16': torch.bfloat16, 'fp32': torch.float32}
+arg_to_torch_dtype = {
+    'fp16': torch.float16, 'bf16': torch.bfloat16, 'fp32': torch.float32, "e5m2": torch.float8_e5m2, "e5m2fnuz":
```

Comment:
these non-fnuz types are for nv backend or future AMD GPUs?

Reply:
I am just trying to support all the fp8 types that are supported by torch. What is your opinion regarding the fp8 types here?